src/main/g8/README.md

```diff
@@ -9,6 +9,11 @@ Sources in the sample:
 * `UserRegistry.scala` -- the actor which handles the registration requests
 * `JsonFormats.scala` -- converts the JSON data from requests into Scala types and from Scala types into JSON responses
 
+## Configure the resolver
+
+> [!TIP]
+> The Akka dependencies are available from Akka’s secure library repository. To access them you need to use a secure, tokenized URL as specified at https://account.akka.io/token.
```

Comment:
Somewhat weird to call it a tip, when it is in fact a requirement.

Reply:
I will just remove the `> [TIP]` part.